schemas/context/viewability-details.schema.json

```diff
@@ -0,0 +1,102 @@
+          "description":
+          "The number of vertical pixels of the window the ad is displayed in, measured at runtime.",
+          "minimum": 0
+
+        },
+        "xdm:viewportWidth": {
+          "title": "Viewport Width",
+          "type": "integer",
+          "description":
+          "The number of horizontal pixels of the window the ad is displayed in, measured at runtime.",
+          "minimum": 0
+        },
+        "xdm:adHeight": {
+          "title": "Ad Height",
+          "type": "integer",
+          "description":
+          "The number of vertical pixels of the player, measured at runtime. This can be larger than the size of the ad if the player has extra controls or thumbnails.",
+          "minimum": 0
+        },
+        "xdm:adWidth": {
+          "title": "Ad Width",
+          "type": "integer",
+          "description":
+          "The number of horizontal pixels of the player, measured at runtime. This can be larger than the size of the ad if the player has extra controls or thumbnails.",
+          "minimum": 0
+        },
+        "xdm:playerVolume": {
+          "title": "Player Volume Percentage",
+          "type": "integer",
+          "description":
+          "The player volume percentage, measured at runtime where 0 is muted and 100 is maximum volume.",
+          "minimum": 0,
+          "maximum": 100
+        },
+        "xdm:measurementEligible": {
+          "title": "Viewability Measurement Eligible",
+          "type": "boolean",
+          "description":
+          "Whether or not the ad was eligible to viewability measurement. An ad is eligible if the unit has a supported creative format and tag type."
+        },
+        "xdm:implementationDetails": {
+          "title": "Viewabiltity implementation details",
+          "type": "object",
+          "$ref": "https://ns.adobe.com/xdm/context/implementationdetails",
+          "description":
+          "The name and version of the library instrumented to measure viewability metrics."
+        },
+        "xdm:viewable": {
+          "title": "Viewable Ad",
+          "type": "boolean",
+          "description":
+          "Display ads are considered viewable when at least 50% of the ad is visible for at least 1 second. Video ads are considered viewable when at least 50% of the ad is visible while the video is playing for at least 2 consecutive seconds. All viewable ads are measurable, because you can't confirm that an ad has met the criteria for viewability unless it can be measured."
+        },
+        "xdm:activeWindow": {
+          "title": "Active Window",
+          "type": "boolean",
+          "description":
+          "Indicates whether ad was shown on active window on users' device."
+        },
+        "xdm:percentViewable": {
+          "title": "Percent Viewable",
+          "type": "integer",
+          "description":
+          "Percent of pixels in the ad deemed viewable at measurement time.",
```

Comment:
What about non-pixel-based media?

Reply:
IAB and MRC guidelines are only defined for pixel-based media. Following industry standard guidelines for consistency.

Reply:
See previous comment on HTML, CSS and SVG

Reply:
If this is a percentage, can you specify min:0, max:100, please?  Otherwise,  I am good.

Reply:
The max and min values should be there already under the description field.